File: src/test/java/net/kigawa/kutil/unit/dummy/Unit4.java
package net.kigawa.kutil.unit.dummy;

import net.kigawa.kutil.unit.annotation.*;
import net.kigawa.kutil.unit.dummy.parent.*;

@Dependency(value = Unit1.class)
@Dependency(value = Unit2.class)
@Unit
public class Unit4 extends AbstractUnit implements UnitInterface1, AllUnitInterface
{
    public Unit4()
    {
    }
}

File: kunit/src/jvmTest/kotlin/net/kigawa/kutil/kunit/dummy/Unit4.kt
package net.kigawa.kutil.kunit.dummy

import net.kigawa.kutil.kunit.api.annotation.Kunit
import net.kigawa.kutil.kunit.dummy.parent.UnitInterface1
import net.kigawa.kutil.kunit.dummy.parent.UnitOneToFourInterface
import net.kigawa.kutil.unit.dummy.parent.AbstractUnit
import kotlin.test.Ignore

@Kunit
@Ignore
class Unit4 : AbstractUnit(), UnitInterface1, UnitOneToFourInterface

